src/pages/CurrenciesListPage.jsx

```diff
@@ -27,6 +41,25 @@ function CurrenciesList() {
         onChange={(event) => setSelectedValue(event.target.value)}
       />
       <div>Value {selectedValue}</div>
+
+      <TableContainer>
+        <Table>
+          <Thead>
+            <Th>Currency</Th>
+            <Th>Value</Th>
+            <Th>Change</Th>
+          </Thead>
+          <Tbody>
+            {currenciesChange.map(({ names, value, change }) => (
+              <Tr key={names + value + change}>
+                <Td key={names}>{names}</Td>
+                <Td key={names + value}>{value}</Td>
+                <Td key={names + change}>{change}</Td>
```

Comment:
While rendering lists, it's enough to pass the key to the container, if it re-renders, all children are re-rendered too.
```suggestion
                <Td>{names}</Td>
                <Td>{value}</Td>
                <Td>{change}</Td>
```

Reply:
done, thanks :)